main.lua

```diff
@@ -250,6 +253,17 @@ local function start(...)
       options.tls.ca = certs.caCertsDebug
     end
 
+    if argv.args.ca or virgo.config['caProvided'] then
+      if argv.args.ca ~= null then
+        fileName = argv.args.ca
+      end
+      if virgo.config['caProvided'] ~= null then
+        fileName = virgo.config['caProvided']
+      end
+
+      options.tls.ca = io.input(fileName):read("*a") -- read the whole file
```

Comment:
Using the IO library blocks the entire thread. In this case, that is fine, but a better option would be to use `fs.readFileSync(filepath)`. fs.readFileSync(filepath) is still blocking, but uses the luvit framework.

Reply:
Thanks Ryan! I have not been doing Lua for quite a while and just grabbed the first thing I found on the internet. I will push this change out shortly.